tests/postgres_tests/test_aggregates.py

```diff
@@ -72,6 +74,70 @@ def test_empty_result_set(self):
                     )
                     self.assertEqual(values, {'aggregation': expected_result})
 
+    def test_default_argument(self):
+        AggregateTestModel.objects.all().delete()
+        tests = [
+            (ArrayAgg('char_field', default=['<empty>']), ['<empty>']),
+            (ArrayAgg('integer_field', default=[0]), [0]),
+            (ArrayAgg('boolean_field', default=[False]), [False]),
+            (BitAnd('integer_field', default=0), 0),
+            (BitOr('integer_field', default=0), 0),
+            (BoolAnd('boolean_field', default=False), False),
+            (BoolOr('boolean_field', default=False), False),
+            (JSONBAgg('integer_field', default=Value('["<empty>"]')), ['<empty>']),
+            (StringAgg('char_field', delimiter=';', default=Value('<empty>')), '<empty>'),
+        ]
+        for aggregation, expected_result in tests:
+            with self.subTest(aggregation=aggregation):
+                # Empty result with non-execution optimization.
+                with self.assertNumQueries(0):
+                    values = AggregateTestModel.objects.none().aggregate(
+                        aggregation=aggregation,
+                    )
+                    self.assertEqual(values, {'aggregation': expected_result})
+                # Empty result when query must be executed.
+                with self.assertNumQueries(1):
+                    values = AggregateTestModel.objects.aggregate(
+                        aggregation=aggregation,
+                    )
+                    self.assertEqual(values, {'aggregation': expected_result})
+
+    def test_convert_value_deprecation(self):
+        AggregateTestModel.objects.all().delete()
+        queryset = AggregateTestModel.objects.all()
+
+        msg = (
+            'From Django 5.0, ArrayAgg() will return None instead of an empty '
+            'list if there are no rows. Pass default=None to opt into the new '
+            "behavior and silence this warning or default=Value([]) to retain "
+            'the existing behavior.'
+        )
+        with self.assertWarnsMessage(RemovedInDjango50Warning, msg):
+            queryset.aggregate(aggregation=ArrayAgg('boolean_field'))
```

Comment:
We ca re-use class variables:
```suggestion
        with self.assertWarnsMessage(RemovedInDjango50Warning, ArrayAgg.deprecation_msg):
            queryset.aggregate(aggregation=ArrayAgg('boolean_field'))
```